tests/providers/amazon/aws/hooks/test_glacier.py

```diff
@@ -47,25 +47,21 @@ def test_retrieve_inventory_should_return_job_id(self, mock_conn):
         assert job_id == result
 
     @mock.patch("airflow.providers.amazon.aws.hooks.glacier.GlacierHook.get_conn")
-    def test_retrieve_inventory_should_log_mgs(self, mock_conn):
+    def test_retrieve_inventory_should_log_mgs(self, mock_conn, caplog):
         # given
```

Comment:
Do we want to remove the `given/when/then` comments in the very few places they still exist while we're at it?

Reply:
Right now main target just migrate to `pytests`.
It is good point to change something from the past 😉
To be honest a lot of test need to rewrite later anyway

Reply:
Yeah, every time I try do add some new code I find a dozen older bits I'd love to redo in the process.  Like why are we mocking get_conn in every test instead of making one fixture to do it?  I'll take care of that one at some point.  Can't fix it all at once :+1: